pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
 section: docs
 ---
 
-The release notes discuss the changes impacting users and implementers:
+### Introduction
 
+Just as your schemas evolve over time, the JSON Schema specification does as well. This guide provides useful information to make upgrading your schemas across versions of the specification a breeze.
+
+Learn how to use our migration guides and tooling ([AlterSchema](https://alterschema.sourcemeta.com/)) to bring your work with you.
+
+### Keywords Overview
+
+Here is a comprehensive overview to get you started.
```

Comment:
comment: I find this sentence incomplete as an introduction to the table. What type of info does the table contain? 

suggestion: "This table provides an overview of keyword changes and removals in JSON Schema specifications from 2019-09 to the latest draft."

Reply:
Noted and applied.